docs/network/interfaces_and_networks.md

```diff
@@ -22,7 +22,7 @@ Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network
 
 ## Networks
 
-Network backends are configured in `spec.networks`. A network must have
+Networks are configured in `spec.networks`. A network must have
```

Comment:
Since the examples now depict `VirtualMachine` objects the correct path is:
```suggestion
Networks are configured in `spec.template.spec.networks`. A network must have
```

Reply:
done